docs/pages/identity-governance/entra-id/manual-installation.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Manual Entra ID integration configuration
+description: Describes how to manually set up Entra ID for Teleport Entra ID integration.
+labels:
+ - how-to
+ - identity-governance
+---
+
+This guide shows manual Entra ID configuration steps to set up Teleport Entra ID integration.
```

Comment:
It might be helpful to add one more sentence to the intro that cross-links to the other guide.

Something like:

> For instructions on the guided installation, see [other guide] instead.